test/integration/rounds.js

```diff
@@ -25,6 +26,7 @@ const randomUtil = require('../common/utils/random');
 const queriesHelper = require('../common/integration/sql/queriesHelper.js');
 const localCommon = require('./common');
 
+global.constants = _.cloneDeep(global.constants);
```

Comment:
can you move this logic to `test/setup.js`, in there we are already doing `global.constants = testContext.config.constants;`,  you can modify the logic in setup to cater the constants behavior for the tests.